docs/studio/cli/team.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Team
+description: Manage and inspect information about your Studio teams.
+keywords:
+  - CLI
+  - command line
+  - studio
+---
+
+## `studio team <subcommand>`
+
+Use these commands to interact with your Studio teams, including listing teams associated with your accounts, creating new teams, and adding members to teams.
+
+### `ls [identifier]`
+
+List all of your teams and underlying projects tied to your Studio account, optionally, passing the team ID.
+
+| Argument       | Type     | Default | Description                         |
+| -------------- | -------- | ------- | ----------------------------------- |
+| `[identifier]` | `string` | --      | (Optional) the team's ID, if known. |
+
+### `create <name> [personalTeamId]`
+
+Create a new team by defining the new name.
+
+| Argument | Type     | Description      |
+| -------- | -------- | ---------------- |
+| `<name>` | `string` | The team's name. |
+
+Optionally, you can pass a personal team ID flag or invite users to the team after creation.
+
+| Option             | Type     | Default | Description                                                     |
+| ------------------ | -------- | ------- | --------------------------------------------------------------- |
+| `--personalTeamId` | `string` | --      | The personal unique team ID (via the `studio team ls` command). |
+| `--invites`        | `string` | --      | Comma separated list of emails to be invited to the team.       |
+
+### `add [team] [user]`
```

Comment:
i also wasnt able to get this to work. I’m not sure which one of these user values are possible, but I think all would be nice to be able to use (email, username, or wallet address). And I tried the team ID instead of team name as well.

```
studio team add --team collabs --user buchhlz2@gmail.com
studio team add --team collabs --user tabledan_friend_2
studio team add --team collabs --user 0xD3C952a5F153Af91a11bEa91DA2Dc2dD87f59Cef
studio team add --team 65048d24-96a8-49b4-8b84-28b636383c87 --user buchhlz2@gmail.com
studio team add --team 65048d24-96a8-49b4-8b84-28b636383c87 --user tabledan_friend_2
studio team add --team 65048d24-96a8-49b4-8b84-28b636383c87 --user 0xD3C952a5F153Af91a11bEa91DA2Dc2dD87f59Cef
```